mypy/build.py

```diff
@@ -1759,9 +1764,10 @@ def __init__(self,
             try:
                 path, follow_imports = find_module_and_diagnose(
                     manager, id, self.options, caller_state, caller_line,
-                    ancestor_for, root_source)
+                    ancestor_for, root_source, temporary)
```

Comment:
I'd use `quick=temporary` here since it's an optional flag. (Though maybe more of the args should use keyword=value then?

Reply:
> Though maybe more of the args should use keyword=value then?

I would do this for `quick` only, because it is a bit "more special" than other.